### DISCUSSION TITLE
更新1290后，我无法更换房间

### DISCUSSION THREAD

**Discussion by lovemeeeeee12314** [Monika-After-Story/MonikaModDev#9606] [general - translations]
我的房间都还在，但是对话中没有“can we go somewhere else”这个对话了。我现在只能待在之前钢琴房间中

**Comment by Wingdinggaster656** [comment]
I used the translator.
I'm guessing your `1290` is a reference to `version 0.12.9`. I'm sorry if there was a mistake.

So, a certain pool topic missed?
Considering that you are using Chinese, did you use a Chinese translation? Many translations are known to have many issues.
